zetachain-cctx/.github/workflows/zetachain-cctx.yml

```
@@ -0,0 +1,93 @@
+      (needs.lint.result == 'success' || needs.lint.result == 'cancelled')
+    uses: ./.github/workflows/_docker-build-push.yml
+    with:
+      service-name: zetachain-cctx
+  publish_types_package:
+    name: Publish types package
+    uses: './.github/workflows/npm-publisher.yml'
+    needs: docker
+    if: needs.docker.outputs.tag-value != ''
+    secrets: inherit
+    with:
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)